caffe2/core/typeid.cc

```diff
@@ -58,11 +61,11 @@ namespace {
 // for unintializied blob. You should not use this struct yourself - it is
 // intended to be only instantiated once here.
 struct UninitializedTypeNameRegisterer {
-  UninitializedTypeNameRegisterer() {
-    gTypeNames()[0] = "nullptr (uninitialized)";
-  }
+    UninitializedTypeNameRegisterer() {
+      gTypeNames()[TypeId::uninitialized()] = "nullptr (uninitialized)";
+    }
 };
 static UninitializedTypeNameRegisterer g_uninitialized_type_name_registerer;
 
 } // namespace
-} // namespace caffe2
+}
```

Comment:
We do have a code practice to end "}" with a namespace quote - so kindly add namespace here.

Reply:
ok added it. On a side note: We should consider enforcing correctness of these comments by a linter or, maybe actually better, removing this practice. The namespace refactoring we're planning to do in the future will break a lot of these if they aren't checked, and wrong comments are worse than no comments.